projects/cmake/DownloadHebiCpp.cmake

```diff
@@ -1,7 +1,7 @@
 # Used to download the C++ API - this should not be used directly.
 cmake_minimum_required(VERSION 3.0)
 
-set(HEBI_CPP_VERSION "2.0.1")
+set(HEBI_CPP_VERSION "2.1.0")
 set(HEBI_CPP_FILE_NAME "hebi-cpp-${HEBI_CPP_VERSION}.tar.gz")
 set(HEBI_CPP_LIB_SHA256 "9930dc65b543510bd211a98ba493482ddb9841afa57ae1f3fb958c6d35a975c7")
```

Comment:
SHA256 definitely isn't the same across two versions...

Reply:
Yup Daniel's right, I believe it needs to be switched to:

91201014ecda682b499c8008fa27446de779fc49ed7abd6827a102c235e7b569